libraries/AP_HAL_PX4/bxcan.h

```diff
@@ -150,21 +49,6 @@ namespace bxcan {
 #define GPIO_CAN2_TX    GPIO_CAN2_TX_2
 #endif
```

Comment:
Why weren't these gone too? If these aren't defined we have a problem somewhere below (in PX4Firmware or PX4Nuttx) and we should give an error if they aren't defined. That is already done in HAL_PX4/CAN.cpp but only for *_RX*, it should be for *_TX* too.